### DISCUSSION TITLE
Firmware update behaviour( flashing green and blue) Raspberry doesn't open

### DISCUSSION THREAD

**Discussion by Batselot** [PiSupply/PiJuice#778] [Q&A]
Hey there,

I have pressed update firmware from pijuice GUI and the device got stuck in a flashing blue and green state. What's the problem and how can I fix this? This issue is continues for a hour now.

**Comment by Batselot** [comment]
For some reason, it worked well. I think it had do with something adapter's power. After a while, it got fixed but I don't know the real reason.

**Reply by tvoverbeek** [reply]
probably a power problem. The blue/green flashing is when the firmware is restarting and initializing. Normally only a few seconds.
